.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+ name: kube2py
+ namespace: mateapp
+ labels:
+   app: kube2py
+spec:
+ selector:
+   matchLabels:
+     app: mate-server
+ template:
+   metadata:
+     labels:
+       app: mate-server
```

Comment:
The label 'app: mate-server' in the pod template should match the label used in the selector. Consider changing it to 'app: kube2py' to ensure consistency and proper functioning of the DaemonSet.